src/components/shmoke.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+<div id='s' class="fl ma2 tc b--solid bg-light-green shadow-5 db pa3"> 
+        <h2>🌲 SHMOKE up FELLA 🌲 </h2>
+        <img class="snoop" @click="shmoke_weed"  :src = getimageUrl() />
+</div>     
+
+</template>
+
+
+<script>
+import { METHODS } from 'http';
+import eventBus from '../eventBus';
+export default {
+    data(){
+        return {
+            shmoking: [
+                {
+            shmoked:true, 
+            name:"shmokie",
+            run: () => {
+                eventBus.$emit('achievement-unlocked' , 'We only smoke quads')
+                eventBus.$emit('big-kush-energy')
+                this.$modal.show('dialog', {
+              title: `<h1> SHMOKING GANJA BIG BOI , ENJOY A 30 🍆 BOOST </h1>`,
+              buttons:[{
+                  title: "puff puff 🅱IGGA"
```

Comment:
😎 puff puff baby

Reply:
no 🅱IGGA"?

Reply:
🅱️ igga is cool too I leave it up to you 👍